### DISCUSSION TITLE
Can't find any kernel to select either in local or remote

### DISCUSSION THREAD

**Discussion by ZhenxuanZhang-Jensen** [microsoft/vscode-jupyter#9505] [Questions and Answers] [answered]
I don't know what's going on, I can select the correct in the morning and the evening, everything has just gone, now, I can' select the kernel, even I can run the python in local and it's the conda env, can somebody help? thanks

**Comment by rchiodo** [comment, marked as answer]
Can you go to this window and copy the contents here?

![image](https://user-images.githubusercontent.com/19672699/160433017-9bb6d924-1da7-47b1-9d11-cc1015d756a3.png)

That should give us an idea why you're not getting any kernels listed.

**Reply by ZhenxuanZhang-Jensen** [reply]
![image](https://user-images.githubusercontent.com/65934516/160433580-58fdf2e0-122d-4689-8d3d-192ce5a64189.png)

**Reply by rchiodo** [reply]
Looks to me like your kernel is dying on startup. The python interpreter we're using to run the kernel is failing to start. 

Your jupyter extension is over a month old. You might try one or more of:
- Upgrading VS code and the jupyter extension to the latest stable release
- Installing VS code insiders and using the jupyter extension prerelease (we've changed how we start kernels a little bit recently)
- Reinstall your higgs-dna environment. It's the one that is crashing.

**Reply by ZhenxuanZhang-Jensen** [reply]
I reinstall my higgs-dna env, upgrade the vscode and use the jupyter extension prerelease version, but it seem the new problem here is that I can't select my higgs-dna env, but I can conda activate this and run with this env
![image](https://user-images.githubusercontent.com/65934516/160553706-6b65f433-8b24-4e1c-a9a3-9474d6febb14.png)

**Reply by ZhenxuanZhang-Jensen** [reply]
well, I reload the windows then it just work, I'm not sure, maybe jupyter need to sync something, anyway, thank you so much for help, I think upgrade the vscode itself really help.
